sale_margin_percentage/models/sale_order.py

```diff
@@ -1,61 +0,0 @@
-from odoo import api, fields, models
-
-
-class SaleOrder(models.Model):
-    _inherit = "sale.order"
-
-    margin_percentage = fields.Float(
```

Comment:
I think we need a migration script here, it is a stored field.

Reply:
done